descwl_shear_sims/tests/test_gen_tanwcs.py

```diff
@@ -27,15 +30,18 @@ def test_gen_tanwcs_dither():
         ra=20 * galsim.degrees,
         dec=-10 * galsim.degrees)
     origin = galsim.PositionD(x=10, y=11)
-    wcs = gen_tanwcs(
+    psf_origin = galsim.PositionD(x=10, y=11)
```

Comment:
Let's make this center different so that we can test that the two origins are not swapped.